R/run_app.R

```diff
@@ -1,29 +1,41 @@
 #' Run the Shiny Application
 #'
-#' @param ... arguments to pass to golem_opts.
-#' See `?golem::get_golem_options` for more details.
+#' @param app_ver a "global" variable that is passed to several modules &
+#'   reports which details the installed package version when not specified.
+#'   It can be overwritten to include a specific version name as a text string.
+#' @param login_note a text string to display underneath the auth screen's login
+#'   button, provided to guide users
+#' @param ... arguments to pass to golem_opts. See `?golem::get_golem_options`
+#'   for more details.
 #' @inheritParams shiny::shinyApp
 #'
 #' @export
 #' @importFrom shiny shinyApp
 #' @importFrom golem with_golem_options
 run_app <- function(
-  onStart = initialize_raa,
+  onStart = initialize_raa(),
   options = list(),
   enableBookmarking = NULL,
   uiPattern = "/",
+  app_ver = paste(packageVersion("riskassessment")),
+  login_note = HTML('<em>Note:</em> To log in for the first time, use either the admin user:
+                        <u>admin</u> or the non-admin user: <u>nonadmin</u>, both with
+                        password <u>QWERTY1</u>.'),
```

Comment:
I prefer handling this in the function body itself. It also allows the opportunity to only show the `login_note` if the credentials database has not been created yet.

[run_app.txt](https://github.com/pharmaR/risk_assessment/files/10015742/run_app.txt)